main.go

```diff
@@ -1,16 +1,73 @@
 package main
 
 import (
+	"bytes"
+	"html/template"
+	"io/ioutil"
 	"log"
+	"net/http"
 	"os"
 
 	"github.com/apex/gateway"
+	"github.com/gin-contrib/static"
 	"github.com/gin-gonic/gin"
+	"github.com/gin-gonic/gin/binding"
+	"github.com/markbates/pkger"
 )
 
+type contactDetails struct {
+	Name 				string `json:"name"`
+	DesignationValue 	string `json:"designation_value"`
+	Business 			string `json:"business"`
+	CurrentWebsiteUrl 	string `json:"current_website_url"`
+	HelpValue 			string `json:"help_value"`
+	Reason 				string `json:"reason"`
+	Email 				string `json:"email"`
+	Message 			string `json:"message"`
+}
+
+func contactDetail(c *gin.Context) {
+	var input contactDetails
+
+	err := c.ShouldBindWith(&input, binding.JSON)
+	if err != nil {
+		c.Abort()
+		c.JSON(http.StatusBadRequest, gin.H{"message": err.Error()})
+		return
+	}
+
+	emailHtml, err := pkger.Open("templates/email-template.html")
+	if err != nil {
+		panic(err)
+	}
+
+	emailFile, err := ioutil.ReadAll(emailHtml)
+	if err != nil {
+		panic(err)
+	}
+
+	emailTemplate := template.Must(template.New("email-template.html").Parse(string(emailFile)))
+
+	var templateBuffer bytes.Buffer
+
+	err = emailTemplate.ExecuteTemplate(&templateBuffer, "email-template.html", input)
+
+	if err != nil {
+		log.Fatal(err)
+		return 
+	}		
+
+	c.JSON(http.StatusOK, input)
+}
+
+
 func main() {
     router := gin.Default()
 
+	router.Use(static.Serve("/", static.LocalFile("./vue-frontend/dist", false)))
```

Comment:
We are not going to serve vue front-end with go anymore, so maybe we can remove this.